src/geovista/cache/__init__.py

```diff
@@ -42,7 +42,7 @@
 """Environment variable to override :mod:`pooch` cache manager path."""
 
 GEOVISTA_DATA_VERSION: str = os.environ.get("GEOVISTA_DATA_VERSION", DATA_VERSION)
-"""Environment variable to override default ``geovista-data`` version."""
+"""Environment variable to override default :attr:`DATA_VERSION`."""
```

Comment:
I would prefer to combine both the old and the new docstring, but that violates line-length limits. Both `GEOVISTA_DATA_VERSION` an `DATA_VERSION` definitely need to be documented as they are important user tools, but I felt it was important to make their relationship clear - the current API documentation reads quite confusingly.

Reply:
@trexfeathers All of the public facing `goevista` environment variables e.g., `GEOVISTA_x`, will be documented in a separate reference section ... which is next on my hit list.

Hopefully that should clear up matters somewhat and we can cross-reference to those environment variable definitions from where ever they're used 👍